cpp/lib/include/HtmlUtil.h

```diff
@@ -107,4 +107,8 @@ size_t ExtractAllLinks(const std::string &html_document, std::vector<std::string
 std::string StripHtmlTags(const std::string &text_with_optional_tags, const bool replace_entities = true);
 
 
+/** \brief Shorten a text but keep HTML structure. */
+std::string ShortenText(const std::string &html_document, const unsigned max_length);
```

Comment:
size_t would be better because that's what std::string::size returns.

Reply:
OK, i changed it (and the length type inside the function as well)